docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,112 @@
+
+- **image selection**: is a use case only in that the tool selecting the image does not eventually adopt one of the simpler proposals [A](PROPOSAL_A.md) or [E](PROPOSAL_E.md), or has use case that is not addressed by a traditional container runtime pulling the image (and using image manifest metadata).
+
+- **provenance**: a provenance tool can easily keep a database of image metadata based on artifacts alone to curate images available on a system, or similar use cases that require such a database.
+
+### Benefits
+
+- A compatibility interest group is free to namespace and define whatever metadata annotations their tooling needs. For example, if a graph schema is required to describe relationships between attributes, the reference URL can be a known metadata attribute. If documentation is desired, this is possible too.
+- The simple structure supports the most basic of use cases (key/value pair checking) to more advanced (graph-based scheduling).
+
+## Image Compatibility Artifact
+
+In [Proposal B](PROPOSAL_B.md) it is stated that a compatibility artifact should describe CRITICAL requirements for container images to run on a host, which is inferred to mean a limited set that determines compatibility. This proposal takes the stance that we don't yet know what that minimal set is, and further, that it may vary depending on the use case for any specific artifact. Instead of declaring a minimum set, this proposal states that the artifact should contain minimum requirements for compatibility but can contain more that MAY be used by plugins to not just determine compatibility but optimal usage. The artifact serves as a source of metadata about container requirements, and it is on the level of the plugins or associated tooling to decide which subset should be used for a given use case. Arguably, this set will change. More discussion and research is required to understand what the line between critical (for the image to run, compatibility) and running optimally.
+
+### Artifact Manifest
+
+This artifact MAY have a manifest that references it with the media type that is specific to the compatibility interest group. For example, metadata for node feature discovery may take the following form:
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.oci.image-compatibility.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1+json",
```

Comment:
I think this is the same case as commented by @sudo-bmitch here https://github.com/opencontainers/wg-image-compatibility/pull/16#pullrequestreview-1980153519

Reply:
yep definitely happy to tweak this to that suggestion!

@mfranczy I have pooooowerrrr!! :joy:

Reply:
Yup, same concern definitely exists for this. Making a spec that consists of other project's specifications, rather than having them make their own artifact, doesn't feel right to me.

Reply:
This was removed.